### PR TITLE
Shell script for building on SteamOS via distrobox

### PR DESCRIPTION
**Pull request by dwu11**
Per our conversation on discord. The default image is Fedora but I think you could probably replace it if you feel more comfortable with another distro.

### CODE REVIEW

**Comment by ALEZ-DEV** [comment]
> The default image is Fedora but I think you could probably replace it if you feel more comfortable with another distro.

No that's okay, I'm currently on Fedora 40 so I'm okay with that.

Just little suggestion, when the installation of the script terminate, create a `run.sh` file (or something similar) locally that you can use to directly open the launcher, like that people can add it to Steam (as a non-steam game) instead of re-enter in distrobox to run the launcher everytime they want to play the game (and this script will be used more by non tech savvy people, so we have to make it the more simple possible to use)

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sudo steamos-readonly disable
+curl -s https://raw.githubusercontent.com/89luca89/distrobox/main/install | sudo sh -s -- --prefix /usr
+sudo steamos-readonly enable
+xhost +si:localuser:$USER >/dev/null
+distrobox create && distrobox enter
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+sudo dnf install pkg-config openssl-devel
+sudo dnf install llvm-devel
+sudo dnf install gcc-c++.x86_64
```
Comment (ALEZ-DEV) [inline build.sh]: We don't need to install Winetricks? Does it use the Wintricks installed directly from the host?

Comment (dwu11) [reply build.sh]: I didn't need it in order to compile.

Comment (ALEZ-DEV) [reply build.sh]: I think so, but you need it in order to install the fonts and dependecies for the game

**Comment by ALEZ-DEV** [comment]
I made some edit to the `build.sh` to build it on steam deck and any other distro
I will make the documentation for it tomorrow

**Pull request merged**